### PR TITLE
Refactor/cars

### PR DESCRIPTION
**Pull request by SrPabvliss**

### CODE REVIEW

File: src/features/vehicles/presentation/components/forms/new-edit-form.vue
Comment (devjaes) [inline src/features/vehicles/presentation/components/forms/new-edit-form.vue:1]: Refactor

File: src/shared/components/filter-search.vue
Comment (devjaes) [inline src/shared/components/filter-search.vue:1]: Use constants

File: src/shared/composables/use-pagination.ts
Comment (devjaes) [inline src/shared/composables/use-pagination.ts:1]: si

File: src/features/vehicles/presentation/components/forms/new-edit-form.vue
Comment (SrPabvliss) [reply src/features/vehicles/presentation/components/forms/new-edit-form.vue:1]: maybe an interation of a new constant FORM_FIELDS and image component as a new component?

File: src/shared/components/filter-search.vue
Comment (SrPabvliss) [reply src/shared/components/filter-search.vue:1]: Be more specific

File: src/shared/composables/use-pagination.ts
Comment (SrPabvliss) [reply src/shared/composables/use-pagination.ts:1]: beso*

**Pull request merged**